### ISSUE TITLE
Change MCP1703-3.3 to something more powerful

### ISSUE THREAD

**Issue by enjoyneering** [dsaltas/WiFi-WebRadio#4]
Hi, dsaltas

According to this studies: [ESP8266 killing itself](https://www.ondrovo.com/a/20170205-esp-self-destruct/) & [Determining the ESP8266 power consumption](https://www.ondrovo.com/a/20170207-esp-consumption/). The ESP8266 uses about 70 mA with frequent short surges (1 μs at 400 mA). In addition, during startup, it draws 430 mA for 40 ms. The MCP1703-3.3 can provide only 250mA. This is what happens on the 3.3V rail on your board.

![MCP1703-33_KaRadio](https://user-images.githubusercontent.com/10623649/67095460-f67e6880-f183-11e9-90d6-b14e861499b2.jpg)

Huge 10Hz peaks around 600 mV if you use the XC6206-3.3. Same story with MCP1703-3.3, but peaks are two times smaller. Ka-Radio is unstable. Small 10uF x 6.3v tantalum capacitor drop emissions to 150mV & 40mV.  WeMos Mini has ME6211 LDO in SOT23-**5** and has peaks of about 40mV with an additional 10uF tantalum capacitor.

Can you change the reference design to accept 47uF tantalum capacitors & 500mA SOT23-**5** LDOs, such as ME6211, RT9193 or RT9013?

I've used XC6206-1.8 for 1.8v rail. It is cheap, easy to find on eBay & it works fine. Would be nice if you add it to the BOM as a replacement option.

Thank you.

**Comment by dsaltas** [comment]
Hi,
Thanks for your feedback,

I have never experienced any power issues and instabilities with the Ka-Radios I have build and tested, although I mostly feed them with 3,3V as the ESP8266 is a 3,3V device.

I will definitely take a look if I can change the footprint of 3,3V LDO to SOT23-5 package as there are many 500mA capable LDOs out there such as MIC5219 or your recommendations.
I will also try to change footprint of filtering capacitor to 1210 package like this 10TPE47MAZB.
It is 47uF/10V tantalum.

In the meantime you can use an ME6211 as it comes in SOT23-3 package. It's a direct drop-in replacement.

Best regards.

**Comment by enjoyneering** [comment]
Hi, dsaltas

The ME6211 in SOT23-3 package is hard to find on Ebay and Aliexpress. Looking forward to changes and thank you for the awesome board.

**Comment by enjoyneering** [comment]
Forgot to mention, the [VS10XX AppNote: Connecting analog outputs](http://www.vlsi.fi/fileadmin/app_notes/vs10XXan_output.pdf) on page 4 is suggesting to add RC-combo to GBUF pin even if you don't use GBUF as output. It is protecting GBUF pin and VS1053 from static electricity.

**Comment by dsaltas** [comment]
Hi,
I have already added this RC-combo to GBUF output pin in a newer revision board (r1b3) that I had built for myself. I never uploaded the newer revision, mostly because I removed the resistors divider from RXD pin. I am using 3,3V so I didn't need the 5V tolerance. I had also increased the values of the output capacitors for better bass response. Apart from these changes, both boards are pretty much the same. You can find the newer revision on hardware directory.

**Comment by enjoyneering** [comment]
ESP8266 **unofficially** 5v tolerant. The CEO of Espressif Systems confirmed it in the tweeter. I've connected many times 5v I2C to GPIO4 & GPIO5.

![ESP8266_5v_tolerant](https://user-images.githubusercontent.com/10623649/67121813-6c052b80-f1ba-11e9-8d13-90fae67869b9.jpg)

Calculation of output capacitor sizes is given on page 8. I changed mine to 100uF, this is more than enough for 10Kohm amplifier input (do not plan to use it with headphones).

![cut_off_req_VS1053](https://user-images.githubusercontent.com/10623649/67122394-b89d3680-f1bb-11e9-9a75-ac9de7d7c72f.jpg)
